src/renderer/Components/RecevingPackage/index.jsx

```diff
@@ -45,9 +54,54 @@ function ReceivingPackage() {
     navigate("/");
   };
 
-  const handleReceiveResult = (event) => {
+  const handleReceiveResult = async (event) => {
```

Comment:
```suggestion
  const handleReceivePackage = async (event) => {
```
마찬가지로 Result 라는 표현이 모호한것 같습니다. 소포를 받고 실행하는 내용이 있으므로 이런 제안을 남겨봅니다!

Reply:
저는 실행 결과를 받아온다는 부분에서 Result라는 표현도 알기 쉽다고 생각했는데 저희 프로그램의 컨셉적인 측면에서 생각해보면 Package라는 표현이 더 적절한것같아 수정 반영하였습니다👍